server/app/services/applicant/question/ApplicantQuestion.java

```diff
@@ -129,6 +129,14 @@ public String getQuestionText() {
     return repeatedEntity.map(r -> r.contextualize(text)).orElse(text);
   }
 
+  /**
+   * Get the question text that should be used for screen readers. We add a styled asterisk, which
+   * should be read out to screen readers.
+   */
+  public String getQuestionTextForScreenReader() {
+    return isOptional() ? getQuestionText() : getQuestionText() + " *";
```

Comment:
Does screen reader reads "asterisk" when processing '*' ? is that convention among screenreader users and they understand what it means?